cmd/catalyst-node/catalyst-node.go

```diff
@@ -906,57 +928,60 @@ func (ac *PlaybackAccessControl) cachePlaybackAccessControlInfo(playbackID, pubK
 
 	var maxAge, stale int64 = 0, 0
 	if len(cacheControlDirectives) == 2 {
-		maxAge, err = strconv.ParseInt(strings.Split(cacheControlDirectives[0], "=")[1], 10, 32)
+		maxAge, err = strconv.ParseInt(strings.Split(cacheControlDirectives[0], "=")[1], 10, 64)
 		if err != nil {
-			return err
+			return 0, 0, 0, err
 		}
 
-		stale, err = strconv.ParseInt(strings.Split(cacheControlDirectives[1], "=")[1], 10, 32)
+		stale, err = strconv.ParseInt(strings.Split(cacheControlDirectives[1], "=")[1], 10, 64)
 		if err != nil {
-			return err
+			return 0, 0, 0, err
 		}
 	}
```

Comment:
I believe all cache-control parsing errors should be non-fatal. If there's an error parsing the header, we just proceed with no cache or force a default cache here like 5 minutes (which is what we're using anyway)